README.md

```diff
@@ -44,6 +44,7 @@ Coding happens all the time and is encouraged. We just recognize there is a poin
 | 64    | [SDK3 Field-Level Encryption](rfc/0064-sdk3-field-level-encryption.md) | David N.     | ACCEPTED |
 | 69    | [KV Error Map V2](rfc/0069-kv-error-map-v2.md)                         | Brett L.     | ACCEPTED |
 | 76    | [KV Subdoc Replica Reads](rfc/0076-subdoc-replica-reads.md)            | Charles D.   | ACCEPTED |
+| 77    | [Zone-Aware Replica Reads](rfc/0077-zone-aware-replica-reads.md)       | Sergey       | ACCEPTED |
```

Comment:
77 is couchbase2 RFC (which should be in here, admittedly)

Reply:
Oh I actually did add it before - https://github.com/couchbaselabs/sdk-rfcs/pull/135/files - it's just impossible to get 3 reviews for this repo..

Reply:
I just followed procedure, and couchbase2 did not claim its number in README

Reply:
Well - it has been trying to claim it for 3 months...  It's just very hard to get things merged into this repo.

Reply:
Anyway, appreciate you've resolved this now, so resolving this conversation.